### PR TITLE
Added Captcha related tests

### PR DESCRIPTION
**Pull request by BlackRider97**

### CODE REVIEW

**Comment by BlackRider97** [comment]
@bombardier-gif Adding you to review the PR

**Comment by bombardier-gif** [comment]
Thank you so much for bringing stability to this script!
Note: I had updated the captcha code to use gif instead of png.. based on some inputs from someone. Could you refresh the code?

**Comment by srmk1** [comment]
Thanks so much for making it more stable, looks good!

File: src/utils.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 OTP_PUBLIC_URL = "https://cdn-api.co-vin.in/api/v2/auth/public/generateOTP"
 OTP_PRO_URL = "https://cdn-api.co-vin.in/api/v2/auth/generateMobileOTP"
 
-WARNING_BEEP_DURATION = (1000, 2000)
+WARNING_BEEP_DURATION = (1000, 5000)
```
Comment (BlackRider97) [inline src/utils.py:16]: changed beep duration is it is helpful if you are away from key board

**Comment by BlackRider97** [comment]
> 
> 
> Thanks so much for making it more stable, looks good!

Updated with latest changes

File: src/utils.py (hunk 1 above)
Comment (bombardier-gif) [reply src/utils.py:16]: +1

**Pull request merged**